packages/expo-firebase-analytics/src/Analytics.ts

```diff
@@ -0,0 +1,198 @@
+import { UnavailabilityError } from '@unimodules/core';
+
+import ExpoFirebaseAnalytics from './ExpoFirebaseAnalytics';
+
+import parseConfig from './parseConfig';
+
+/**
+ * Get the bundled Google Services config file.
+ * This is useful for debugging if your app was built properly.
+ */
+export function getBundledGoogleServicesConfig(): null | { [key: string]: any } {
+  return ExpoFirebaseAnalytics.app || null;
+}
+
+/**
+ * Similar to `firebase.initializeApp()` on web but works to start a native Firebase app while the app is running.
+ * This can be used to test the native iOS Firebase app in the Expo client.
+ * This method should not be used in production, instead the app should be bundled with the native Google Services files via the `app.json`.
+ *
+ * @param googleServices Platform specific Google Services file for starting a Firebase app during runtime
+ */
+export async function initializeAppDangerously(googleServices: {
+  [key: string]: any;
+}): Promise<void> {
+  // @ts-ignore
+  if (global.__DEV__ !== true) {
+    console.warn('initializeAppDangerously should only be used in dev mode');
+  }
+  if (!ExpoFirebaseAnalytics.initializeAppDangerously) {
```

Comment:
Without `|| {}` in `ExpoFirebaseAnalytics.ts` or `if (!ExpoFirebaseAnalytics || !ExpoFirebaseAnalytics.init…)` this (and other similar checks) will throw a not customized TypeError if the module is unavailable